File: RxNetWorkLibrary/src/main/java/io/reactivex/network/util/OkioUtils.java
package io.reactivex.network.util;

import java.io.IOException;

import okhttp3.internal.cache.DiskLruCache;

/**
 * by y on 20/09/2017.
 */

public class OkioUtils {


    public static void abort(DiskLruCache.Editor editor) {
        if (editor != null) {
            try {
                editor.abort();
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
    }

    public static void snapshot(DiskLruCache.Snapshot snapshot) {
        if (snapshot != null) {
            snapshot.close();
        }
    }
}

File: RxNetWorkLibrary/src/main/java/io/reactivex/network/util/OkioUtils.kt
package io.reactivex.network.util

import okhttp3.internal.cache.DiskLruCache
import java.io.IOException

/**
 * by y on 20/09/2017.
 */

object OkioUtils {


    fun abort(editor: DiskLruCache.Editor?) {
        if (editor != null) {
            try {
                editor.abort()
            } catch (e: IOException) {
                e.printStackTrace()
            }

        }
    }

    fun snapshot(snapshot: DiskLruCache.Snapshot?) {
        snapshot?.close()
    }
}
